metanovaverse/chain.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "metanovaverse",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.blockxnet.com/",
```

Comment:
is the blockx website a false copy-paste artifact?

Reply:
Updated the website url